### ISSUE TITLE
Release model

### ISSUE THREAD

**Issue by feldsam** [mailcow/mailcow-dockerized#971]
Hello, I see, that there are no releases and also changes are merged to master too early, so master branch is no stable as should be.

I think, that this is not small project and also user base grows, so we should think about that, start making releases, write change logs, prepare beta releases for testing new functionality, etc..

**Comment by mkuron** [comment]
This has already been discussed in #734, #340, and some others. The dev branch should be considered beta-quality and the master branch is for productive use. Errors happen and as long as @andryyy is the only one who spends a lot of time on the dev branch, any errors that he doesn't catch will almost certainly make it into master. For all of us, Mailcow is only a side project, so we just don't have the resources to take it to the next level in terms of stability and testing.

Basically, the advice for productive use is to not pull the latest master immediately after a major merge from dev, but to wait a few hours or a day. That way, @andryyy or someone else will have time to catch the bug. Follow the issue tracker to see whether there have been any major problem reports since the last merge.

**Comment by Braintelligence** [comment]
Or you're like me and pull it anyway 😸. @andryyy reacts very fast.
I'm backing up the whole virtual machine before upgrading, though.
